internal/storage/storage_handle_test.go

```diff
@@ -253,6 +254,186 @@ func (testSuite *StorageHandleTest) TestCreateHTTPClientHandle_WithGRPCClientPro
 	assert.Contains(testSuite.T(), err.Error(), fmt.Sprintf("client-protocol requested is not HTTP1 or HTTP2: %s", cfg.GRPC))
 }
 
+func (testSuite *StorageHandleTest) TestCreateHTTPClientHandle_WithReadStallRetry() {
+	testCases := []struct {
+		name                 string
```

Comment:
nit:
```suggestion
		tcname                 string
```

Reply:
Doesn't testCase itself tell the name is for test-case?